src/util/storybook-docs/Related.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from 'react';
+import { DocsContext } from '@storybook/addon-docs/blocks';
+import { linkTo } from '@storybook/addon-links';
+
+export const Related = () => {
+  const context = useContext(DocsContext);
+
+  const related = context.parameters.docs.fds?.related;
+
+  if (!related) return null;
+
+  return (
+    <div className="related">
+      Are you looking for:
```

Comment:
I actually like "Related" copy here better - it's a bit more neutral. To me "Are you looking for" sounds like an indication of a mistake when there might not be one